contrib/lpass-att-export.sh

```diff
@@ -2,35 +2,84 @@
 ##
 ## Usage: lpass-att-export.sh
 ##
-##  - login via lpass login
-##  - run ./lpass-att-export.sh
-##  - check subfolder 'lpass-export'
 ##
 
+set -x
```

Comment:
I think this is a debug artifact hanging around

Reply:
Done!